debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/MethodMetaData.java

```diff
@@ -569,6 +590,11 @@ public UDATA getJitDataResolvePushes() throws CorruptDataException {
 				   12 slots saved integer registers
 				*/
 				return new UDATA(12);
+			} else if (J9ConfigFlags.arch_aarch64) {
+				/* AArch64 data resolve shape
+				   29 integer registers (not saving x29/x30/x31)
+				*/
```

Comment:
Please format comment with `*` beginning the second and subsequent lines.
```
	/* AArch64 data resolve shape
	 * 29 integer registers (not saving x29/x30/x31)
	 */
```

Reply:
Fixed format of the comment.